packages/storage-postgres/prisma/schema.prisma

```diff
@@ -0,0 +1,20 @@
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model WorkflowState {
+  workflowId        String      @id
```

Comment:
We should add the `@db.Uuid` type here so that it's using the right postgres data type